src/webview/js/rewriteHtml.js

```diff
@@ -58,11 +58,37 @@ export const rewriteImageUrls = (auth: Auth, element: Element | Document = docum
   });
 };
 
+/* Rewrite the 'time' elements to make them more readable.
+ *
+ * 1. Changes text to localized human readable date-time.
+ * 2. Adds 'original-text' attribrute, to show original text alert dialog on press.
+ */
+const rewriteTime = () => {
+  const timeElements: NodeList<HTMLTimeElement> = document.querySelectorAll('time');
+  timeElements.forEach(elem => {
+    const timeStamp = elem.getAttribute('datetime');
+    const text = elem.innerText;
+
+    // 'text' and 'timeStamp' are guaranteed to be present.
```

Comment:
This comment, and the code below it, look a little funny to me, like, "This will definitely be here. If it's not...".

I think the check is probably being done to satisfy Flow that these things will, in fact, be present. A brief explanation of that fact would help this make more sense, so it doesn't seem self-contradictory. 🙂